UNRELEASED.md

```diff
@@ -10,3 +10,14 @@
          * Some bug fix, see #124
 
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
+
+### Features
+
+* New errors for the following constant simplification scenarios (see #1191):
+  1. Division by 0
+  1. Mod (%) by 0
+  1. Negative expoents
+  1. Expoents bigger than an Integer
+  1. Expoential operations that would overflow `BigInt`
+
+### Bug Fixes
```

Comment:
We can leave this to be added if/when bug fixes get added during this release period.
```suggestion
```